### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by ahzamir**
## In this pull request I have worked on the To-Do List project structure and made it as requirements:

- I set up the project with webpack.
- I have created an index.html file and wrote my HTML markup there. Create an empty To-Do List placeholder (<div>).
- The index.html file is set as a template using the HTML Webpack Plugin.
- I have created an index.js file and set an array of some simple to-do tasks (array of objects). Each task object contains three keys:
  - description [string].
  - completed [bool].
  - index: [number].
- I have written a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list appears in order of the index values for each task.
- I have created a style.css and set rules for the To-Do List. CSS loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  mode: 'development',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  module: {
```
Comment (Cathella) [inline webpack.config.js:15]: ✅ Webpack is used to set up the project 👍🏻 
✅ It has an HTML file configured in the Webpack config settings as a template using the *HTML Webpack Plugin* 👍🏻

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  {
+    description: 'thirth',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'second',
+    completed: false,
+    index: 1,
+  },
```
Comment (Cathella) [inline src/index.js:24]: ✅ It has an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\] 👍🏻

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @ahzamir 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Professional README.
✅ Used webpack to bundle JavaScript.


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzamir** [comment]
@Cathella thank you very much for your kind feedback and review. 😊

**Pull request merged**